### ISSUE TITLE
ivy: spacemacs//counsel-async-filter throws error

### ISSUE THREAD

**Issue by rgemulla** [syl20bnr/spacemacs#10189]
When running ```spacemacs/search-grep``` or any other ```spacemacs/search-*``` function on the develop branch, the following error is thrown as soon as grep starts to run (i.e., the minimum number of chars have been entered):
```
error in process filter: let: Not enough arguments for format string
```

The error arises in ```spacemacs//counsel-async-filter``` here:
```elisp
        (let ((ivy--prompt
               (format (ivy-state-prompt ivy-last)
                       size)))
```

The reason is that the ivy prompt wants to show the number of the current candidate and the total number of candidates. In particular, ```(ivy-state-prompt ivy-last)``` gives, for example:
```
"%d/%d ag from [c:/WINDOWS/system32/]: "
```

A working fix is to replace those lines by, e.g.,:
```elisp
        (let ((ivy--prompt
               (format (ivy-state-prompt ivy-last)
                       1 size)))
```

**Comment by al3xandru** [comment]
Seeing this issue on master (0.200.13) with `counsel-search-docs` (`SPC h R`)

**Comment by duianto** [comment]
The PR: Fix ivy prompt #12030
has been cherry-picked into the develop branch.
